### PR TITLE
Add proguard rules

### PR DESCRIPTION
**Pull request by CB-RyanMcCormick**

### CODE REVIEW

**Review by chauduyphanvu — APPROVED** [review]

File: VungleAdapter/proguard-rules.pro [hunk 1]
```diff
@@ -19,3 +19,27 @@
...
+
+# Google
+-keep class com.google.android.gms.** { *; }
+-dontwarn com.google.android.gms.**
+
+# GSON
+-keepattributes *Annotation*
+-keepattributes Signature
+# Prevent R8 from leaving Data object members always null
+-keepclassmembers,allowobfuscation class * {
```
Comment (bichenwang) [inline VungleAdapter/proguard-rules.pro:35]: you sure we should allow obfuscation for data objects?

**Review by bichenwang — APPROVED** [review]
lgtm otherwise

**Review by cb-jpadilla — APPROVED** [review]
✅

File: VungleAdapter/proguard-rules.pro (hunk 1 above)
Comment (CB-RyanMcCormick) [reply VungleAdapter/proguard-rules.pro:35]: I haven't seen any problems arise from this, but I guess we'll see if QA finds something I didn't.

**Pull request merged**